UnoCheck/Checkups/XCodeCheckup.cs

```diff
@@ -50,23 +50,7 @@ public override Task<DiagnosticResult> Examine(SharedState history)
 
 				if (selected is not null && selected.Version.IsCompatible(MinimumVersion, ExactVersion))
 				{
-					// customize runner options so the license can be displayed
```

Comment:
The license check should be done first (or very early) since every invocation of `Xcode` or `xcodebuild` might fail if the license is not accepted. That behaviour can differ a bit in every Xcode release (so this might work now but won't in the next release).

Reply:
Updated the license to be validated first